.github/workflows/deploy.yml

```
@@ -0,0 +1,27 @@
+name: Deploy
+
+on:
+  release:
+    types: [created]
```

This should probably be:
```suggestion
    types: [published]
```
so that saving a draft release doesn't trigger the workflow.